web/src/subscriber_api.ts

```diff
@@ -1,14 +1,19 @@
 import * as channel from "./channel";
 import * as people from "./people";
-
+import type {StreamSubscription} from "./sub_store";
```

Comment:
The blank line you deleted was there to separate the imports from the comment; leave it there.

Reply:
done @andersk . Apologies for some trivial mistakes. I have fixed the spacing after import and edited commit messages.
Thanks for your patience and guidance.